README.md

```diff
@@ -36,10 +36,17 @@ ecosystem are currently rebuilt.
 
 ## Usage
 
-The `oss-rebuild` CLI tool can be used to inspect attestations:
+The `oss-rebuild` CLI tool provides access to OSS Rebuild data:
 
 ```bash
+$ go run github.com/google/oss-rebuild/cmd/oss-rebuild@latest --help
+$ # Alternatively, install the binary locally.
+$ # Just make sure it's on your PATH: https://go.dev/ref/mod#go-install
 $ go install github.com/google/oss-rebuild/cmd/oss-rebuild@latest
+```
```

Comment:
actually wdyt about also including `$ oss-rebuild --help` as a last line here for symmetry?